portfolio/src/main/webapp/index.html

```diff
@@ -6,84 +6,73 @@
+            <br/><br/>
+            The Patronus that I got from one of the Pottermore quizzes is the Tortouiseshell Cat, which is cool because I really like cats. 
+            Unfortunately I do not have any pets but I do have a stuffed animal substitute named Regulus. Regulus looks like a cat but she's actually a toy dog. 
+            Just like a cat I like eating fish, especially salmon and eel. If the food comes in the form of sushi, that's even better.
+            </p>
         </div>
-        
-        <p><strong>Other Pictures</strong></p>
-        <div id="image_label">
-            <a href="/images/WizardsUniteProfile.png">
-                <img src="/images/WizardsUniteProfile.png" width=250 alt="My Wizards Unite Profile">
-            </a>
-            <p>I am a huge Harry Potter fan!</p>
+        <div id="section">
+            <h2>Interesting Projects</h2>
+            <p>
+                Besides the Unity project that I mentioned above, another project that I did that I find particularly interesting is Zing. 
+                Zing is a project that I started with two people over the summer my freshman year when I did an internship at Life Changing Labs in Ithaca. 
+                Life Changing Labs is an entrepreneurship program which matches founders with interns that they work with to build their startup idea. 
+                Zing is a project that brings isolated students on campus together and during this time of quarantine we actually made a variant of Zing called 
+                Quarantine Buddy which pairs users to do a Zoom call together so they don't have to feel as isolated when they are at home. A special 
+                shoutout to my teammate Sam who got the opportunity to be on ABC news with Kelly and Ryan to talk about Quarantine Buddy.
+                <br/><br/>
+                I also worked on a bomb defusal game with the C programming language on a single FRDM MK64F board with my friend. 
+                We used the board's built-in accelerometer to determine orientation values for displaying the different sections of the game. 
+                Each section had a different mini-game to play: Morse code translation, simon says, reaction time, and a last stand game which depended on the results of the other 3 mini-games. 
+                The game uses the LED to display a randomly selected sequence of colors based on the rules of each mini-game and the user uses the pushbuttons to play the game.
+                </p>
         </div>
-        <div id="image_label">
-            <a href="/images/MangoMangoSweets.jpg">
-                <img src="/images/MangoMangoSweets.jpg" width=250 alt="Mango Mango dessert">
-            </a>
-            <p>Yum! I have a sweet tooth.</p>
+        <div id="section" style="display:flex;">
+            <div style="padding-right: 25px; margin-right: 25px; border-right: 5px solid orange;">
```

Comment:
Could you move the style into css file?

Reply:
Added a commit which moves all styles from the html files into the css file.